build.sbt

```diff
@@ -126,18 +126,21 @@ lazy val mimaEnable: List[Def.Setting[_]] = List(
     else Set("org.scalameta" % moduleName.value % previousVersion)
   }
 )
+
 val sharedJVMSettings: List[Def.Setting[_]] = List(
   crossScalaVersions := allScalaVersions
 ) ++ mimaEnable
+
 val sharedJSSettings: List[Def.Setting[_]] = List(
   skipIdeaSettings,
   crossScalaVersions := allScalaVersions.filterNot(_.startsWith("0."))
 )
 val sharedJSConfigure: Project => Project =
   _.disablePlugins(MimaPlugin)
+
 val sharedNativeSettings: List[Def.Setting[_]] = List(
   skipIdeaSettings,
-  crossScalaVersions := scala2Versions
+  crossScalaVersions := allScalaVersions
```

Comment:
I think crossScalaVersions can be now moved to sharedSetting since it's the same for all platforms (the filter in the JS version is a leftover of the Dotty 0.x days)